presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -129,6 +129,7 @@
     private boolean orderByAggregationSpillEnabled = true;
     private boolean windowSpillEnabled = true;
     private boolean orderBySpillEnabled = true;
+    private boolean spillableAccumulatorPreprocessingEnabled = true;
```

Comment:
shall we make it to false? It is safer to rollout in prod

Reply:
I wanted to keep this enabled in code so that it can go though out strict release verification process. That way, we would be able to detect any regression/issues during release verification itself. If we disable it in code and the enable it via config in prod, then we will miss the exhaustive testing coverage that we would have gotten otherwise.  

I am also going to perform exhaustive verifier runs to test this PR but still any additional testing is cherry on top. Let me know what you think and if you have strong opinion for disabling it.

Reply:
lol, don't trust our release process. We don't validate spilling during our release ....